docs/user_guides/fs/storage_connector/creation/snowflake.md

```diff
@@ -51,7 +72,9 @@ Head to the Storage Connector View on Hopsworks (1) and set up a new storage con
 Enter the details for your Snowflake connector. Start by giving it a **name** and an optional **description**.
 
 1. Select "Snowflake" as connector protocol.
-2. Specify the hostname for your account in the following format `<account_identifier>.snowflakecomputing.com`.
+2. Specify the hostname for your account in either the account identifier format
+`<account_identifier>.snowflakecomputing.com` 
+or any of the account URL format `https://<orgname>-<account_name>.snowflakecomputing.com`.
```

Comment:
Same comments above about calling this "account URL" and the `https://` at the beginning of the string.